app/models/user.rb

```diff
@@ -61,10 +61,18 @@ def user
   end
 
   def add_to_organization(organization)
-    organization && members.
-      find_or_create_by(organization: organization) do |member|
-      member.entry_date = DateTime.now.utc
-    end
+    return unless organization
+
+    member = members.where(organization: organization).first_or_initialize
```

Comment:
`members` or `Member`?

Reply:
`members` would be the same as doing `Member.where(user: self, ...`